db/migrate/20170902203116_add_archive_to_need.rb

```diff
@@ -0,0 +1,6 @@
+class AddArchiveToNeed < ActiveRecord::Migration[5.1]
+  def change
+    add_column :needs, :active, :boolean, default: true
```

Comment:
Should it be null: false as well?

Reply:
yeah, I think ideally it should be. Just didn't want to go back and update current records at the same time as the geo stuff.